extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureStorage.java

```diff
@@ -49,7 +57,7 @@ public AzureStorage(
     this.cloudBlobClient = cloudBlobClient;
   }
 
-  public List<String> emptyCloudBlobDirectory(final String containerName, final String virtualDirPath)
+  public List<String> emptyCloudBlobDirectory(String containerName, final String virtualDirPath)
```

Comment:
we're not changing the containerName in this function are we? I think it's confusing if one parameter is marked as final and the other isn't when both are in fact final